lib/rex/proto/dns/resolver.rb

```diff
@@ -84,8 +84,15 @@ def initialize(config = {})
           raise ResolverArgumentError, "Option #{key} not valid"
         end
       end
+
       self.static_hostnames = StaticHostnames.new(hostnames: static_hosts)
-      self.static_hostnames.parse_hosts_file
+      begin
+        self.static_hostnames.parse_hosts_file
```

Comment:
Is it possible to ignore the individual entries that break? 👀

i.e. If an invalid entry is the first line of the /etc/hosts file, we could ignore that individual host and continue to parse the rest of the file with `127.0.0.1 localhost` etc